RobotCode/src/main/java/frc/robot/constants/ShooterMountConstants.java

```diff
@@ -3,7 +3,9 @@
 public class ShooterMountConstants
 {
 
-	public static final double TICKS_IN_ONE_DEGREE = 4096 / 360; // Update to reflect gear ratios
+	// Note: The .0's are necessary, because otherwise java thinks they are integers-
```

Comment:
Convert to a javadoc. Also the .0s are not necessary. Java can implicitly cast integers to doubles because no data is lost (you cannot go from doubles to ints implicitly).

Reply:
From testing, this seems to be incorrect. Java will do integer division regardless of the type (yes, we did test this, and it was a problem) if we do not have the '.0' or append a 'd' at the end.